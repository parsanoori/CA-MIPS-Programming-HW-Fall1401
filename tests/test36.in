3
572459833
145431879
967206917