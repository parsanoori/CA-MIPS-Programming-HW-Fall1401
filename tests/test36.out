145431879
572459833
967206917